pep-0008.txt

```diff
@@ -850,6 +850,30 @@ names.
 In some fonts, these characters are indistinguishable from the
 numerals one and zero.  When tempted to use 'l', use 'L' instead.
 
+ASCII Compatibility
```

Comment:
I think this could be shorter, containing just a reminder that in the stdlib, https://www.python.org/dev/peps/pep-3131/#policy-specification applies.